src/controllers/upsert/config-vars.js

```diff
@@ -0,0 +1,32 @@
+const core = require('@actions/core');
+
+const heroku = require('../../heroku');
+
+class ConfigVarsStep {
```

Comment:
Might require a move to TS but it'd be cool to have these extend an interface that requires `checkPrereqs` and `run`.

Reply:
Great call — done! See https://github.com/readmeio/heroku-review-app-action/pull/35/commits/e8bf4032e49d6c2823543598a19a51d0c803c9a9